src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -26,7 +29,7 @@ namespace ProductConstructionService.DependencyFlow;
 internal abstract class PullRequestUpdater : IPullRequestUpdater
 {
 #if DEBUG
-    private static readonly TimeSpan DefaultReminderDelay = TimeSpan.FromMinutes(3);
+    private static readonly TimeSpan DefaultReminderDelay = TimeSpan.FromMinutes(1);
```

Comment:
I am not sure this won't make the INT scenario tests flaky. But we can try definitely.

Reply:
Ah I didn't know we rolled out INT in DEBUG, I can revert it, I just used it to speed up local tests

Reply:
Ah, maybe not, maybe we never changed it. So leave it like this then.

Maybe this eventually should be a config setting.